File: JavaScript/countdigit.js
function countdigit(arr){
    let digit=0
    let letter=0
    for(let i=0; i<arr.length;i++){
        let ch=arr[i]
        if(ch>="0" && ch<="9"){
            digit++
        }
        if(ch>="a" && ch<="z"){
            letter++
        }
        if(ch>="A" && ch<="Z"){
            letter++
        }
    }
    return {digit, letter}
}
console.log(countdigit("fhskjdkioe8457324"))

File: python/countdigit.py
def countdigitletter(arr):
    digit=0
    letter=0
    for i in arr:
        if "0"<=i<="9":
            digit+=1
        elif ("a"<=i<="z") or ("A"<=i<="Z"):
            letter+=1
    return {digit, letter}
print(countdigitletter("hjgdasfjkiorqrbcxnzqotre53453463ak"))